Mathlib/Analysis/Normed/Group/Continuity.lean

```diff
@@ -151,8 +203,8 @@ theorem ContinuousAt.norm' {a : α} (h : ContinuousAt f a) : ContinuousAt (fun x
 theorem ContinuousAt.nnnorm' {a : α} (h : ContinuousAt f a) : ContinuousAt (fun x => ‖f x‖₊) a :=
   Tendsto.nnnorm' h
 
-@[to_additive (attr := fun_prop) ContinuousAt.enorm]
-lemma ContinuousAt.enorm' (h : ContinuousAt f a) : ContinuousAt (‖f ·‖ₑ) a := Tendsto.enorm' h
+@[fun_prop]
+lemma ContinuousAt.enorm (h : ContinuousAt f a) : ContinuousAt (‖f ·‖ₑ) a := Tendsto.enorm' h
```

Comment:
Why is this not `to_additive` anymore? Is this confusingly not needing `SeminormedGroup E` from the context? Can you move this to an earlier section then?

Reply:
This was generalised to speak about any ContinuousENorm: I have moved it (and its relatives) to `Basic.lean`.